myhpi/wagtail_markdown/minutes_extensions.py

```diff
@@ -100,10 +100,10 @@ def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
         self.patterns = [
             (
-                r"\|enter\|\(([0-9:]+)\)\(([\w,\ ]+)\)(\((?P<mean_of_participation>.*?)\))?",
+                r"\|enter\|\(([0-9:]+)\)\(([^\)\(]+)\)(\((?P<mean_of_participation>.*?)\))?",
                 self.enterify,
             ),
-            (r"\|leave\|\(([0-9:]+)\)\(([\w,\ ]+)\)", self.leavify),
+            (r"\|leave\|\(([0-9:]+)\)\(([^\)\(]+)\)", self.leavify),
```

Comment:
If the roles are accepted, this would need to be:
```suggestion
            (r"\|leave\|\(([0-9:]+)\)\(((?:[^\)\(]+(?:\([^\)]+\))?)+)", self.leavify),
```

Reply:
I think I will replace enter with leave in your version ^^